l3build-file-functions.lua

```diff
@@ -214,22 +214,21 @@ end
 -- Copy files 'quietly'
 function cp(glob, source, dest)
   local errorlevel
-  for i,_ in pairs(tree(source, glob)) do
-    local source = source .. "/" .. i
+  for p_rel,p_cwd in pairs(tree(source, glob)) do
```

Comment:
the values in `tree(source, glob)` are already relative to the CWD.
`p_rel` may rename in `p_src`?

Reply:
Why `p_`? More generally the naming seems off. `p_cwd` looks as if it's the path of the current working directory, but it should be the path to the source file. So it should include some reference to `source` in it's name. IMO `p_src` would be a good fit there since paths are normally assumed to be (absolute or) relative to the CWD, so it's just the "normal" path to the source.

So maybe keeping `p_rel` and replacing `p_cwd` with `p_src`?